content/spin/managing-templates.md

```diff
@@ -1,24 +1,132 @@
 title = "Managing Templates"
 template = "spin_main"
-date = "2023-01-01T00:00:00Z"
+date = "2022-03-14T00:22:56Z"
 enable_shortcodes = true
 [extra]
-keywords = "install"
+url = "https://github.com/fermyon/spin/blob/main/docs/content/template-managing.md"
 
 ---
+- [Installing Templates](#installing-templates)
+  - [Installing From the Spin Git Repository](#installing-from-the-spin-git-repository)
+    - [Installing as Part of a Specific Language SDK](#installing-as-part-of-a-specific-language-sdk)
+  - [Installing From a Specific Branch](#installing-from-a-specific-branch)
+  - [Installing From a Local Directory](#installing-from-a-local-directory)
+- [Viewing Your Installed Templates](#viewing-your-installed-templates)
+- [Uninstalling Templates](#uninstalling-templates)
+- [Upgrading Templates](#upgrading-templates)
+  - [Upgrading Templates From a Local Directory](#upgrading-templates-from-a-local-directory)
+- [Next Steps](#next-steps)
 
-## Managing Templates
+Templates are a Spin tool for scaffolding new applications and components. You can use them via the `spin new` and `spin add` commands. For more information about creating applications with templates, see [Writing Spin Applications](writing-apps).
 
-Spin templates are designed to allow application developers to create the skeleton of an application or component via the `spin` command. You can start learning about how to manage Spin templates by reading [the template section](/common/cli-reference#templates) of the Spin Command Line Interface (CLI) documentation. If you are interested, there is documentation about [creating Spin templates](/spin/template-authoring) for advanced users and alternate use cases. For now, let’s focus on managing and using Spin’s existing templates.
+## Installing Templates
 
-### Housekeeping
+> This section covers general principles for installing templates. For information about installing templates for specific languages, see [Writing Spin Applications](writing-apps).
 
-Please go ahead and [install](/spin/install) (or update) Spin. The Spin version information is available via the following command:
+To install templates, use the `spin templates install` command. You can install templates from a Git repository, or while [authoring templates](template-authoring) you can install them from a local directory.
+
+### Installing From the Spin Git Repository
+
+To install templates from the Spin Git repository, run `spin templates install --git`.
+
+<!-- @nocpy -->
+
+```bash
+$ spin templates install --git https://github.com/fermyon/spin
+```
+
+The above command installs _all_ templates in the repository.
+
+#### Installing as Part of a Specific Language SDK
```

Comment:
This seems at odds with a "managing templates" page.  There's nothing different about "language SDK" templates or how they're installed; I would instead put this info under the language guides.  At most I would say something like "Language SDKs often ship templates in their repositories; see the relevant language guide to find out where to get its templates."

Reply:
Thanks @itowlson ... updating now ...